backend/src/api/FeedController.ts

```diff
@@ -59,20 +64,19 @@ export default class FeedController {
         this.router.post('/feed/all', validate(feedSubscriptionsSchema), (req, res) => this.feedAll(req, res));
         this.router.post('/feed/posts', validate(feedPostsSchema), (req, res) => this.feedPosts(req, res));
         this.router.post('/feed/watch', validate(feedWatchSchema), (req, res) => this.feedWatch(req, res));
-        this.router.post('/feed/sorting', saveFeedSortingLimiter, (req, res) => this.saveFeedSorting(req, res));
+        this.router.post('/feed/sorting', saveFeedSortingLimiter, validate(feedSortingSchema), (req, res) => this.saveFeedSorting(req, res));
     }
 
-    async saveFeedSorting(request: APIRequest<FeedSortingSaveRequest>, response: APIResponse<FeedSortingSaveResponse>) {
+    async saveFeedSorting(request: APIRequest<FeedSortingSaveRequest>, response: APIResponse<Record<string, unknown>>) {
         if (!request.session.data.userId) {
             return response.authRequired();
         }
 
         const userId = request.session.data.userId;
 
         try {
-            await this.userManager.saveFeedSorting(request.body['site-id'], request.body['feed-sorting'] as FeedSorting, userId);
+            await this.userManager.saveFeedSorting(request.body['site'], request.body['feedSorting'] as FeedSorting, userId);
```

Comment:
It's valid way to access field, but seems a bit odd to me, let's use
`request.body.site`, `request.body.feedSorting`